pkg/build/client/clients.go

```diff
@@ -21,24 +21,24 @@ type BuildConfigUpdater interface {
 	Update(buildConfig *buildapi.BuildConfig) error
 }
 
-// OSClientBuildConfigClient delegates get and update operations to the OpenShift client interface
-type OSClientBuildConfigClient struct {
-	Client osclient.Interface
+// ClientBuildConfigClient delegates get and update operations to the OpenShift client interface
+type ClientBuildConfigClient struct {
+	Client builds.Interface
 }
 
-// NewOSClientBuildConfigClient creates a new build config client that uses an openshift client to create and get BuildConfigs
-func NewOSClientBuildConfigClient(client osclient.Interface) *OSClientBuildConfigClient {
-	return &OSClientBuildConfigClient{Client: client}
+// NewClientBuildConfigClient creates a new build config client that uses an openshift client to create and get BuildConfigs
+func NewClientBuildConfigClient(client builds.Interface) *ClientBuildConfigClient {
+	return &ClientBuildConfigClient{Client: client}
 }
 
 // Get returns a BuildConfig using the OpenShift client.
-func (c OSClientBuildConfigClient) Get(namespace, name string, options metav1.GetOptions) (*buildapi.BuildConfig, error) {
-	return c.Client.BuildConfigs(namespace).Get(name, options)
+func (c ClientBuildConfigClient) Get(namespace, name string, options metav1.GetOptions) (*buildapi.BuildConfig, error) {
```

Comment:
these should die eventually.  Doesn't need to be now.

Reply:
@bparees i'll make a follow up issue for you :)